components/Tab.scss

```diff
@@ -1,17 +1,20 @@
 .tab {
-  display: flex;
   position: relative;
-  align-items: center;
-  cursor: pointer;
-  border-left: solid 1px #888;
-  background: var(--tabs-bg) var(--bg-gradient);
-  padding: 0 10px;
+  // Logical property to allow border flip on toggle button
+  border-inline-end: solid 1px var(--tabs-border);
   font-weight: 600;
   font-size: 12pt;
 
-  &.active {
-    border-color: transparent;
-    background: var(--highlight);
+  button {
+    all: unset;
+    display: flex;
+    align-items: center;
+    background: var(--tabs-bg) var(--bg-gradient);
+    padding: 0 10px;
+  }
+
+  &.active button {
+    background: var(--accent);
```

Comment:
I reworked the DOM to fix the borders. The contrast was poor and one went missing on `hover` and `active`

<img width="429" alt="Screenshot 15" src="https://github.com/user-attachments/assets/2d033d90-4098-4589-b680-5dc4109a7ef0">

<img width="429" alt="Screenshot 16" src="https://github.com/user-attachments/assets/eeecef02-4c8d-4c37-8f4f-0b10f09d3723">